lib/utility/mock.dart

```diff
@@ -0,0 +1,6 @@
+import 'package:MobileOne/providers/share_provider.dart';
```

Comment:
wrong place wrong content

this file should contain all

MonMock extends Mock implements Maclasse {}

and be located in the test folder